packages/app-builder/src/models/data-model.ts

```diff
@@ -187,9 +187,12 @@ export function getDataTypeIcon(dataType?: DataType): IconName | undefined {
     case 'Timestamp':
       return 'schedule';
     case 'String':
+    case 'String[]':
```

Comment:
I think I did not get your point here.
Is `getDataTypeIcon` the issue ? or the fact it's present in the file `data-model.ts` ? or something else ?

Reply:
Yea it's more the principle that "data model" for me represents the user's data model (as in the schema of his tables), rather than the types of data that are possibly handled in intermediate AST steps. Two things that are very related but one is included in the other.
But that's fine now really